src/main/java/com/mesabrook/ib/blocks/gui/telecom/GuiMSACBootScreen.java

```diff
@@ -0,0 +1,125 @@
+package com.mesabrook.ib.blocks.gui.telecom;
+
+import com.mesabrook.ib.net.ClientSoundPacket;
+import com.mesabrook.ib.util.handlers.PacketHandler;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumHand;
+
+import java.io.IOException;
+
+
+public class GuiMSACBootScreen extends GuiPhoneBase
+{
+    private int timerToNextScreen = 0;
+    private int fadeAnimationTimer = 0;
+    private String currentTexture;
+
+    public GuiMSACBootScreen(ItemStack phoneStack, EnumHand hand)
+    {
+        super(phoneStack, hand);
+        fadeAnimationTimer++;
+    }
+
+    @Override
+    protected String getInnerTextureFileName()
+    {
+        if(fadeAnimationTimer == 0)
+        {
+            currentTexture = "app_screen_no_bar.png";
+        }
+        if(fadeAnimationTimer == 50)
+        {
+            currentTexture = "msac/boot_screen_msac_0.png";
+        }
+        if(fadeAnimationTimer == 100)
+        {
+            currentTexture = "msac/boot_screen_msac_1.png";
+        }
+        if(fadeAnimationTimer == 150)
+        {
+            currentTexture = "msac/boot_screen_msac_2.png";
+        }
+        if(fadeAnimationTimer == 200)
+        {
+            currentTexture = "msac/boot_screen_msac_3.png";
+        }
+        if(fadeAnimationTimer == 250)
+        {
+            currentTexture = "msac/boot_screen_msac_4.png";
+        }
+        if(fadeAnimationTimer == 300)
+        {
+            currentTexture = "msac/boot_screen_msac_5.png";
+        }
+        if(fadeAnimationTimer == 400)
+        {
+            currentTexture = "msac/boot_screen_msac_6.png";
+        }
+        if(fadeAnimationTimer == 500)
+        {
+            currentTexture = "msac/boot_screen_msac_1.png";
+        }
+
+        if(fadeAnimationTimer > 550)
+        {
+            fadeAnimationTimer = 100;
+        }
+
+        return currentTexture;
+    }
+
+    @Override
+    protected boolean renderControlBar() {
+        return false;
+    }
+
+    @Override
+    protected boolean renderTopBar() {
+        return false;
+    }
+
+    @Override
+    public void initGui()
```

Comment:
Unnecessary override

Reply:
That void is overridden in almost every other Minedroid GUI class lol

Reply:
Right, so the purpose of this method is to construct your buttons and textboxes and stuff. Since we're just showing various sets of pictures based on the current timer, there's nothing to construct.

You're right - overriding won't cause any problems, but it can affect readability if we're debugging something or add one more thing in case we decide to get rid of `initGui` in the future. Overall, it doesn't cause harm, but isn't good practice.

Reply:
I hear that lol, I'll keep it in mind for the future.